app/models/account.rb

```diff
@@ -1,3 +1,4 @@
 class Account < ApplicationRecord
     validates :name, presence: true
+    has_many :users
```

Comment:
It's important to specify what happens with its associations when an model is deleted.
`has_many :users, dependent: :destroy`